app.js

```diff
@@ -34,10 +34,11 @@ app.use(helmet());
 app.use(cors());
 app.use(xss());
 
-app.get('/', (eq, res) => {
-  res.send('<h1>Jobs API</h1><a href="/api-docs">Documentation</a>')
-})
+// app.get('/', (eq, res) => {
+//   res.send('<h1>Jobs API</h1><a href="/api-docs">Documentation</a>')
+// })
```

Comment:
Avoid committing commented-out code! Prefer to just delete it; it will live on in version history!

Reply:
I am commenting this code so I can refer it in future to see how I went through the course.

Reply:
I understand that there are valid motivations for commenting out code, but it's a bad practice! You already have an outstanding tool for looking back at past versions of your code and it's called git! There are many ways you could find something like this in git history after you delete it from your codebase. For example, if you do `git diff <previous commit I'd> HEAD`, you can see all the changes between the latest version and that previous commit. If you have enough information to know the specific commit the change occured in (you presumably would in this case), then you can just look at that commits changes.

Either way, committing commented out code is a bad practice, and getting better with git to the extent that you can find that old code when you need it will pay you huge dividends.